src/components/modals/profileSetting/ProfileSetting.jsx

```diff
@@ -0,0 +1,203 @@
+import PropTypes from 'prop-types';
+import { FaSpinner, FaFileUpload } from 'react-icons/fa';
+import { Formik, Form } from 'formik';
+import * as Yup from 'yup';
+import Dropzone from 'react-dropzone';
+import { useState } from 'react';
+import FormField from '../../FormField/FormField';
+import Modal from '../Modal/Modal';
+
+const ContactSchema = Yup.object().shape({
+  firstName: Yup.string().min(2).max(10),
+  lastName: Yup.string().min(2).max(10),
+  title: Yup.string().min(2).max(24),
+});
+function ProfileSetting({ open, setOpen }) {
+  // eslint-disable-next-line no-unused-vars
+  const [newProfilePic, setProfilePic] = useState('');
+  // eslint-disable-next-line no-unused-vars
+  const [newCoverPic, setCoverPic] = useState('');
```

Comment:
Is the setter not being used? You can remove them btw.

Reply:
This is what we gonna use to store the pic locally before dispatching them with the form before updating our profile.
Let me know if you want me to remove them for now.